### ISSUE TITLE
Suggested simplification of some apparently useless settings

### ISSUE THREAD

**Issue by nekohayo** [Drive4ik/simple-tab-groups#885]
STG's settings page has many useful settings, but it seems to me like these ones are useless (and therefore complicate your code a bit), and they clutter up the view which makes for a slightly less nice user experience navigating the settings page:

* "Show content menu on tabs": I don't know why anybody would want to turn this off. This is pretty much an essential feature, and it takes only one line in a UI (Firefox's contextual menus) without being intrusive.
* "Show content menu on links": same thing

Regarding the "enable dark theme" preference... is there a way the extension could simply detect if Firefox's global dark theme is set? That way it could be controlled automatically by things like https://addons.mozilla.org/en-US/firefox/addon/automatic-dark/

**Comment by Drive4ik** [comment]
I agree about the fact that there are a lot of settings, and that now the name of the addon does not quite fit because of the greater number of settings 😄 Only I'm not planning to change the name of the addon yet))
Removing "unnecessary" settings is also a bad option, because a lot of settings I made at the request of users, and a lot of people need them. Statistics on the use of settings unfortunately do not.
The settings just need to be more intelligently organized and divided into groups/tabs. I'll look into it sometime. If anyone has a desire to help with this - send me a list or pictures of the desired settings window and/or their grouping.
About the automatic theme change - I've already done it, will be in the next release. c21cdb53bca088b0b43fe5835225336559eec818
